indico/modules/events/registration/templates/display/registration_summary.html

```diff
@@ -1,3 +1,47 @@
+{% macro render_registration_info(registration) %}
+    <table>
+        {% for section in registration.registration_form.sections -%}
+            <tr>
+                <td class="regform-done-title">{{ section.title }}</td>
+            </tr>
+            {% for field in section.children -%}
```

Comment:
{% for field in section.children if field.is_field -%}

and get rid of the `if` in the next line